crates/sui-graphql-rpc/src/types/cursor.rs

```diff
@@ -0,0 +1,357 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{fmt, ops::Deref};
+
+use async_graphql::{
+    connection::{CursorType, OpaqueCursor},
+    *,
+};
+use serde::{de::DeserializeOwned, Serialize};
+
+use crate::{config::ServiceConfig, error::Error};
+
+/// Wrap the `OpaqueCursor` type to implement Scalar for it.
+pub(crate) struct Cursor<C>(OpaqueCursor<C>);
+
+/// Connection field parameters parsed into a single type that encodes
```

Comment:
Is there supposed to be more after "encodes"

Reply:
Indeed there is, thanks for spotting!